dockerfiles/dockerfile.simplatform

```diff
@@ -0,0 +1,11 @@
+ARG baseVersion
+ARG dockerRepository
+FROM ${dockerRepository}/galasadev/galasa-base:${baseVersion}

```

Comment:
Again this should be the new base image location which is `FROM ${dockerRepository}/galasa-dev/base-image:${baseVersion}`

Reply:
done